ibl-frontend/frontend-content/src/assets/images/training_pipeline.svg

```diff
@@ -0,0 +1 @@
="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 38px; font-family: &quot;Roboto Condensed&quot;; color: rgb(26, 26, 26); line-height: 1.2; pointer-events: all; font-weight: bold; white-space: normal; overflow-wrap: normal;"><span style="font-size: 38px"><font style="font-size: 38px">. . .</font></span></div></div></div></foreignObject><text x="1331" y="303" fill="#1A1A1A" font-family="Roboto Condensed" font-size="38px" text-anchor="middle" font-weight="bold">. . .</text></switch></g><rect x="465" y="279" width="64" height="48" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 62px; height: 1px; padding-top: 292px; margin-left: 466px;"><div style="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 38px; font-family: &quot;Roboto Condensed&quot;; color: rgb(26, 26, 26); line-height: 1.2; pointer-events: all; font-weight: bold; white-space: normal; overflow-wrap: normal;"><span style="font-size: 38px"><font style="font-size: 38px">. . .</font></span></div></div></div></foreignObject><text x="497" y="303" fill="#1A1A1A" font-family="Roboto Condensed" font-size="38px" text-anchor="middle" font-weight="bold">. . .</text></switch></g><rect x="465" y="505" width="64" height="48" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow: visible; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 62px; height: 1px; padding-top: 518px; margin-left: 466px;"><div style="box-sizing: border-box; font-size: 0px; text-align: center;"><div style="display: inline-block; font-size: 38px; font-family: &quot;Roboto Condensed&quot;; color: rgb(26, 26, 26); line-height: 1.2; pointer-events: all; font-weight: bold; white-space: normal; overflow-wrap: normal;"><span style="font-size: 38px"><font style="font-size: 38px">. . .</font></span></div></div></div></foreignObject><text x="497" y="529" fill="#1A1A1A" font-family="Roboto Condensed" font-size="38px" text-anchor="middle" font-weight="bold">. . .</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Please add a link to the **editable** version of this graphic to make it easier for maintaining changes to it in the future. Seems like the `README.md` would be the best place to include it.

Also, could you include in the `README.md` the link for the google doc/publication that the internal navigator is pointing to that governs what is rendered for that deployment too?

Reply:
> Please add a link to the **editable** version of this graphic to make it easier for maintaining changes to it in the future. Seems like the `README.md` would be the best place to include it.

Which README file would you like me to use? If linking, do you want it publicly available? Is so, should I move it from datajoint google drive or just generate a link to access it from there? Another option is to include the editable version in the assets or some other raw folder. 

> Also, could you include in the README.md the link for the google doc/publication that the internal navigator is pointing to that governs what is rendered for that deployment too?

I actually forgot about this one and started working on it today. I was thinking of just making an html page for that material instead of an iframe to a google doc but if you want to stick to the google doc I can do that too. I have the same question as above for the google doc link, which README to use (also, this documentation is on the internal version of the navigator, which I think is a diff branch?).